Debugging

blootsvoets opened the pull request:
Updated to RADAR-Commons 0.3, which had some changes in the testing code.

fnobilia committed:
Updated common jars

blootsvoets committed:
Use unsafe messaging to communicate with localhost

blootsvoets committed:
Committed the latest commons snapshot

blootsvoets committed:
Update to RADAR-Commons 0.3

blootsvoets committed:
Latest dockerRadar branch

blootsvoets committed:
Using Java 8

fnobilia commented on src/integrationTest/java/org/radarcns/integration/util/ExpectedDocumentFactory.java:
```diff
@@ -50,28 +50,36 @@
             case AVERAGE:
-                return collectors.convertToAvro().getAvg();
+                return collectors.getCollectors().stream()
+                        .map(DoubleValueCollector::getAvg).collect(Collectors.toList());
```
We discussed the use of Java `stream` offline, and we agreed it does not perform in situation where there is parallelism. Is this just a choice to make code more readable?

Note: the comment applies to all other cases of this `switch`

blootsvoets replied:
Yes, just because `convertToAvro` is no longer available... But I see in the dev branch you already implemented a List summation. I can remove this and use your code instead.

fnobilia commented on build.gradle:
```diff
@@ -12,8 +12,8 @@ war {
-targetCompatibility = '1.7'
-sourceCompatibility = '1.7'
+targetCompatibility = '1.8'
+sourceCompatibility = '1.8'
```
The java version was blocked to `1.7` due to a bug in the `Tomcat` docker image, has this bug been solved?

blootsvoets replied:
I've updated the Tomcat image to use Java 8 in RADAR-CNS/RADAR-Docker@e8f1f99, I suppose that would solve the issue?

fnobilia reviewed (commented):
Thank you for integrating the new `RADAR-COMMONS` within the test phase. Very useful!

I am waiting the end of the `Travis` tests before checking the conflicts.

blootsvoets committed:
Merge branch 'dev' into debugging

blootsvoets committed:
A bit simpler notation in ExpectedDataSetFactory

blootsvoets committed:
Slight loop reordering

fnobilia committed:
Merge branch 'dev' into debugging

WIP

fnobilia committed:
Updated tomcat version and docker for e2e test

fnobilia committed:
Fixed integration test in getUserEffectiveTimeFrame

fnobilia committed:
Fixed e2e tests

The new `radar-commons` `0.3` has reintroduced the error in the
conversion from `float` to `double`

fnobilia committed:
Remove avro dependencies

fnobilia committed:
Switched to radar-commons 0.3.1-SNAPSHOT

fnobilia committed:
Updated docker infrastructure

blootsvoets committed:
Use data root in all mock classes

fnobilia committed:
Updated schema version and added getSubject end-point

blootsvoets committed:
Use Gradle for all tests + small changes to config

blootsvoets committed:
Merge branch 'debugging' of github.com:RADAR-CNS/RADAR-RestApi into debugging

blootsvoets committed:
Fixed endToEndTest

fnobilia committed:
Fixed stopRadarStack and added RECEIVED_MESSAGES

fnobilia committed:
Added integration test to Travis

blootsvoets committed:
Updated to latest RADAR-Commons testing code

fnobilia committed:
Added test case for RECEIVED_MESSAGES

blootsvoets committed:
Fixed ExpectedValue bug and local integrationTest

blootsvoets committed:
Use sudo to clean radar stack on linux

blootsvoets committed:
Working integrationTest and endToEndtest (except receivedMessages)

fnobilia committed:
Fixed test for RECEIVED_MESSAGES

fnobilia committed:
Reset docker-compose

fnobilia committed:
Increased test latency
